Submitted By:            Fernando de Oliveira <famobr at yahoo dot com dot br>
Date:                    2013-11-01
Initial Package Version: 2.1
Upstream Status:         Fixed
Origin:                  ArchLinux
Description:             Fixes cookies in libavformat/http.

diff -Naur ffmpeg-2.1.orig/libavformat/http.c ffmpeg-2.1/libavformat/http.c
--- ffmpeg-2.1.orig/libavformat/http.c	2013-10-28 01:58:06.000000000 +0100
+++ ffmpeg-2.1/libavformat/http.c	2013-11-04 11:30:25.146215255 +0100
@@ -95,7 +95,7 @@
 {"multiple_requests", "use persistent connections", OFFSET(multiple_requests), AV_OPT_TYPE_INT, {.i64 = 0}, 0, 1, D|E },
 {"post_data", "set custom HTTP post data", OFFSET(post_data), AV_OPT_TYPE_BINARY, .flags = D|E },
 {"mime_type", "set MIME type", OFFSET(mime_type), AV_OPT_TYPE_STRING, {0}, 0, 0, 0 },
-{"cookies", "set cookies to be sent in applicable future requests, use newline delimited Set-Cookie HTTP field value syntax", OFFSET(cookies), AV_OPT_TYPE_STRING, {0}, 0, 0, 0 },
+{"cookies", "set cookies to be sent in applicable future requests, use newline delimited Set-Cookie HTTP field value syntax", OFFSET(cookies), AV_OPT_TYPE_STRING, {0}, 0, 0, D },
 {"icy", "request ICY metadata", OFFSET(icy), AV_OPT_TYPE_INT, {.i64 = 0}, 0, 1, D },
 {"icy_metadata_headers", "return ICY metadata headers", OFFSET(icy_metadata_headers), AV_OPT_TYPE_STRING, {0}, 0, 0, 0 },
 {"icy_metadata_packet", "return current ICY metadata packet", OFFSET(icy_metadata_packet), AV_OPT_TYPE_STRING, {0}, 0, 0, 0 },
